tests/rptest/transactions/tx_admin_api_test.py

```diff
@@ -302,8 +295,9 @@ def test_delete_non_existent_tid(self):
                     self.admin.delete_partition_from_transaction(
                         error_tx_id, partition["ns"], partition["topic"],
                         partition["partition_id"], partition["etag"])
+                    assert False
                 except requests.exceptions.HTTPError as e:
-                    assert e.response.text == '{"message": "Unexpected tx_error error: {tx::errc::unknown_server_error}", "code": 500}'
+                    assert e.response.status_code == 404
```

Comment:
why wouldn't we like to check the message text too?

Reply:
i think the error code assert is enough as the message is only a hint